### PR TITLE
Change `1` in `one` key to `%{count}`

### LINKED ISSUE #823: Use of %{count} inside the one: form of pluralization rules?
For some translations, the `one:` form hard-codes the value `1`, e.g.:
https://github.com/svenfuchs/rails-i18n/blob/8720fa7050d6683f0b0079069f29bc758ef5ff23/rails/locale/en.yml#L133

But for other translations of the same key, the `one:` form embeds the value of `%{count}`, e.g.:
https://github.com/svenfuchs/rails-i18n/blob/8720fa7050d6683f0b0079069f29bc758ef5ff23/rails/locale/nl.yml#L133

We noticed this because we have a CI test that compares the set of interpolation parameters that appear in the same string, and its assertion that the set should be equal across all of our supported languages triggered a failure.

If I've understood the code correctly, the rules for the following languages actually use the `one:` form for values in the range `0 < x < 2` (for example, `1.5`):  `fr`, `ff`, `kab`, and `lag`.  So, at least for those languages, it would be better to embed `%{count}` than to hard-code `1` or `un`.  For this reason, we make a practice of using `%{count}` in our own code.

What is the overall policy / opinion here?  If I were to prepare a PR updating `rails-i18n/rails/locale/*.yml` to use `%{count}` for the various `one:` cases, would there be any interest in merging it?

(This is tangentially related to https://github.com/svenfuchs/rails-i18n/issues/461)

### PR DESCRIPTION
**Pull request by movermeyer**
### What are you trying to accomplish?

Fixes #823. cc @cejaekl

`one` is used for numbers other than 1.

> A common mistake is to think that "one" is only for only the number 1.
> Instead, "one" is a category for any number that behaves like 1.
> So in some languages, for example, one → numbers that end in "1" (like 1, 21, 151) but that don't end in 11 (like 11, 111, 10311).
- http://cldr.unicode.org/index/cldr-spec/plural-rules

### What approach did you choose and why?

Simple find/replace, then looked through the changes to make sure they were correct.

Find: `one:(.*)1(.*)`
Replace: `one:$1%{count}$2`
Then `git add -up`

### What should reviewers focus on?

🤷 

### The impact of these changes

Correct translations when lookups are done in locales that use `one` for non-1 counts.

### CODE REVIEW

**Comment by pama** [comment]
Do we need to change it for all languages?

I don't think adding an interpolation is the best solution for languages that will always be 1 for `one:`. I would rather see a fix for the languages where "one" is not only for the number 1.

**Comment by movermeyer** [comment]
> I don't think adding an interpolation is the best solution for languages that will always be 1 for `one:`. I would rather see a fix for the languages where "one" is not only for the number 1.

@pama I respectfully disagree. I believe we should change it for all languages.

a) Translation files should be synchronized in the strings that they represent. 
- e.g., If you use a translation management system, it will assume that the strings are the same
- Detecting differences in the "shape" of translations between languages is a good way to automatically detect issues with the translations.

b) There's no harm in having the interpolations.
c) Sometimes the rules change. For example, the rules for [`fr`](https://github.com/unicode-org/cldr/pull/1100/files), [`es`, `it`, `pt`](https://github.com/unicode-org/cldr/pull/1439) recently changed. If you don't use the interpolations, then you need to go through and re-evaluate each time (which we can assume won't happen).

**Comment by pama** [comment]
Your arguments seem reasonable. Thank you.

I noticed that the tests are failing.

**Comment by pama** [comment]
@movermeyer will you be able to address the CI errors? Thanks.

**Comment by movermeyer** [comment]
> @movermeyer will you be able to address the CI errors? Thanks.

@pama Rebased and fixed up. Thanks.

**Comment by sunny** [comment]
Would you mind adding this to the CHANGELOG, as well? 🙏🏻

**Comment by movermeyer** [comment]
> Would you mind adding this to the CHANGELOG, as well? 🙏🏻

@sunny Done.

**Comment by movermeyer** [comment]
> @movermeyer Have you considered requesting the same in the Ruby on Rails repository?

Will do!

File: rails/locale/pt.yml [hunk 1]
```diff
@@ -63,41 +63,41 @@ pt:
...
-        one: quase 1 ano
+        one: quase %{count} ano
         other: quase %{count} anos
       half_a_minute: meio minuto
       less_than_x_seconds:
-        one: menos de 1 segundo
+        one: menos de %{count} segundo
         other: menos de %{count} segundos
       less_than_x_minutes:
         one: menos de um minuto
```
Comment (silva96) [inline rails/locale/pt.yml:82]: You missed this one:

one: menos de %{count} minuto

File: rails/locale/pt-BR.yml [hunk 2]
```diff
@@ -63,41 +63,41 @@ pt-BR:
...
-        one: quase 1 ano
+        one: quase %{count} ano
         other: quase %{count} anos
       half_a_minute: meio minuto
       less_than_x_seconds:
-        one: menos de 1 segundo
+        one: menos de %{count} segundo
         other: menos de %{count} segundos
       less_than_x_minutes:
         one: menos de um minuto
```
Comment (silva96) [inline rails/locale/pt-BR.yml:82]: You missed this one too:

one: menos de %{count} minuto

File: rails/locale/pt.yml (hunk 1 above)
Comment (movermeyer) [reply rails/locale/pt.yml:82]: Good call. I was just using a very simple regex to find these.

I've now written a script to detect differing interpolations between the pluralization context subkeys. Running it, I see that there are 566 similar issues across 71 locales:

```
16 problems in `ar` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.messages.other_than", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length", "errors.template.header"]
11 problems in `bn` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
6 problems in `bs` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years"]
12 problems in `cs` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
13 problems in `da` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "datetime.distance_in_words.x_years", "errors.template.header"]
13 problems in `de` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "datetime.distance_in_words.x_years", "errors.template.header"]
12 problems in `de-AT` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
12 problems in `de-CH` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
12 problems in `de-DE` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
15 problems in `dz` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "datetime.distance_in_words.x_years", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length", "errors.template.header"]
7 problems in `el` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years"]
7 problems in `el-CY` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years"]
1 problems in `en` locale: ["datetime.distance_in_words.less_than_x_minutes"]
1 problems in `en-AU` locale: ["datetime.distance_in_words.less_than_x_minutes"]
1 problems in `en-CA` locale: ["datetime.distance_in_words.less_than_x_minutes"]
1 problems in `en-CY` locale: ["datetime.distance_in_words.less_than_x_minutes"]
1 problems in `en-GB` locale: ["datetime.distance_in_words.less_than_x_minutes"]
1 problems in `en-IE` locale: ["datetime.distance_in_words.less_than_x_minutes"]
1 problems in `en-IN` locale: ["datetime.distance_in_words.less_than_x_minutes"]
1 problems in `en-NZ` locale: ["datetime.distance_in_words.less_than_x_minutes"]
1 problems in `en-TT` locale: ["datetime.distance_in_words.less_than_x_minutes"]
1 problems in `en-US` locale: ["datetime.distance_in_words.less_than_x_minutes"]
1 problems in `en-ZA` locale: ["datetime.distance_in_words.less_than_x_minutes"]
7 problems in `eo` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years"]
2 problems in `et` locale: ["datetime.distance_in_words.almost_x_years", "errors.template.header"]
11 problems in `eu` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
11 problems in `fa` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months"]
13 problems in `fi` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "datetime.distance_in_words.x_years", "errors.template.header"]
10 problems in `fr` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length"]
10 problems in `fr-CA` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length"]
10 problems in `fr-CH` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length"]
10 problems in `fr-FR` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length"]
16 problems in `fy` locale: ["datetime.distance_in_words.about_x_hours.one", "datetime.distance_in_words.about_x_months.one", "datetime.distance_in_words.about_x_years.one", "datetime.distance_in_words.almost_x_years.one", "datetime.distance_in_words.less_than_x_seconds.one", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years.one", "datetime.distance_in_words.x_seconds.one", "datetime.distance_in_words.x_minutes.one", "datetime.distance_in_words.x_days.one", "datetime.distance_in_words.x_months.one", "datetime.distance_in_words.x_years.one", "errors.messages.too_long.one", "errors.messages.too_short.one", "errors.messages.wrong_length.one", "errors.template.header.one"]
2 problems in `gl` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.over_x_years"]
12 problems in `he` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
10 problems in `hi` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
10 problems in `hi-IN` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
9 problems in `id` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months"]
1 problems in `is` locale: ["errors.template.header"]
7 problems in `it` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years"]
6 problems in `it-CH` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years"]
7 problems in `kn` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years"]
8 problems in `lb` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "errors.messages.wrong_length"]
10 problems in `mg` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length"]
4 problems in `ml` locale: ["datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "errors.template.header"]
13 problems in `mr-IN` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length", "errors.template.header"]
2 problems in `ms` locale: ["datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes"]
1 problems in `nb` locale: ["errors.template.header"]
7 problems in `nl` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years"]
15 problems in `oc` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "datetime.distance_in_words.x_years", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length"]
11 problems in `pl` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length", "errors.template.header"]
1 problems in `pt` locale: ["datetime.distance_in_words.less_than_x_minutes"]
1 problems in `pt-BR` locale: ["datetime.distance_in_words.less_than_x_minutes"]
9 problems in `rm` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months"]
7 problems in `ro` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "errors.template.header"]
16 problems in `sc` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds.one", "datetime.distance_in_words.x_minutes.one", "datetime.distance_in_words.x_days.one", "datetime.distance_in_words.x_months.one", "datetime.distance_in_words.x_years.one", "errors.messages.too_long.one", "errors.messages.too_short.one", "errors.messages.wrong_length.one", "errors.template.header.one"]
12 problems in `sk` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
2 problems in `sl` locale: ["datetime.distance_in_words.less_than_x_minutes", "errors.template.header"]
1 problems in `sq` locale: ["datetime.distance_in_words.less_than_x_minutes"]
16 problems in `st` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "datetime.distance_in_words.x_years", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length", "errors.template.header"]
13 problems in `sv` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "datetime.distance_in_words.x_years", "errors.template.header"]
12 problems in `sv-FI` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
13 problems in `sv-SE` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "datetime.distance_in_words.x_years", "errors.template.header"]
3 problems in `sw` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.almost_x_years", "errors.template.header"]
5 problems in `ta` locale: ["datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "errors.messages.too_long", "errors.messages.too_short"]
16 problems in `te` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "datetime.distance_in_words.x_years", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length", "errors.template.header"]
12 problems in `tl` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
14 problems in `tt` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length"]
15 problems in `ug` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length", "errors.template.header"]
15 problems in `ur` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length", "errors.template.header"]
1 problems in `wo` locale: ["datetime.distance_in_words.less_than_x_minutes"]
Total: 566 problems in 71 locales
```

**Pull request merged**